src/launcher.ts

```diff
@@ -70,7 +70,7 @@ class Launcher {
             case Runtime.WebLocal:
                 if (CONFIG.performance) {
                     console.log(`${chalk.gray('[build]')} starting local web build ${chalk.green(build.commit)} multiple times and measuring performance...`);
-                    return this.measure(signal => this.launchWebPerformance(build, signal));
+                    return this.runWebPerformance(build, new AbortController().signal);
```

Comment:
There is a `NO_ABORT_SIGNAL` in this file that you should be using if abort is unsupported. However, now it seems all abort support is removed? If that moved into `vscode-perf` then I suggest to remove aborting from here entirely.

Reply:
Abort is still used in `runWebPerformance` in the case if process exists while downloading the web server.

Reply:
No, with your changes, none of the run-methods use `AbortSignal` anymore that is ever aborted. You introduce `new AbortController().signal` as signal, but that is never dealt with (i.e. never cancelled).

Think of this as `CancellationToken` that always needs a `CancellationTokenSource` that cancels the operation.

Here is the code you had removed that did it:

https://github.com/bpasero/vscode-bisect/blob/6bbc7d203e20b35af3e521b8608b05c12e0b32dc/src/launcher.ts#L123-L124

Reply:
Ok, got it